Libraries/Image/ImageBackground.js

```diff
@@ -57,16 +57,9 @@ class ImageBackground extends React.Component<$FlowFixMeProps> {
   render(): React.Node {
     const {children, style, imageStyle, imageRef, ...props} = this.props;
 
-    // If `no` is specified for `importantForAccessibility`, it will be changed to `no-hide-descendants` because the text inside should not be focused.
```

Comment:
Won't importantForAccessibility no longer work at all on ImageBackgrounds now?

I think we should still have this property, it just should work like normal where no = no and no_hide_descendants = no_hide_descendants.

If this is magically being inherited from somewhere else and it *does* work still, feel free to ignore my comment :)

Reply:
I seem to have misunderstood.
ImageBackgrounds' importantForAccessibility is given to the Image inside, not the Wrapper itself, and it doesn't seem to work because of that.
I will fix this. Thanks！